addons/web/static/src/views/fields/daterange/daterange_field.js

```diff
@@ -52,6 +62,19 @@ export class DateRangeField extends Component {
             },
             () => [this.root.el]
```

Comment:
isn't this line the culprit? Can't we add the value as dependency, to re-configure the picker when it changes?

Reply:
Yes, it s done